src/bin/pgcopydb/schema.c

```diff
@@ -749,7 +749,7 @@ struct FilteringQueries listSourceTablesSQL[] = {
 		"                                on i.indrelid = a.attrelid "
 		"                               and a.attnum = ANY(i.indkey) "
 		"                               and i.indisprimary "
-		"                   where a.attrelid = c.oid "
+		"                   where a.attrelid = c.oid and a.attisdropped = false"
```

Comment:
I think I would prefer the notation `and not a.attisdropped`. Could you update the patch?

Reply:
Done